catalog-info.yaml

```
@@ -0,0 +1,44 @@
+---
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+  name: carbon
+  description: Carbon is an open-source design system built by IBM
+  tags:
+    - carbon
+    - designsystem
+  annotations:
+    github.com/project-slug: carbon-design-system/carbon
+    backstage.io/techdocs-ref: dir:.
+  links:
+    - url: https://carbondesignsystem.com
+      title: Carbon Design System website
+    - url: https://discord.gg/J7JEUEkTRX
+      title: Discord channel
```

Here's a link to the slack channel
```suggestion
  links:
    - url: https://carbondesignsystem.com
      title: Carbon Design System website
    - url: https://ibm-studios.slack.com/archives/C0M053VPT
      title: Internal carbon-design-system slack channel
    - url: https://discord.gg/J7JEUEkTRX
      title: Discord channel
```